packages/frontend/src/components/Explorer/VisualizationCard/ShowTotalsToggle.tsx

```diff
@@ -0,0 +1,26 @@
+import { Label, Switch } from '@blueprintjs/core';
+import { FC } from 'react';
+import { useVisualizationContext } from '../../LightdashVisualization/VisualizationProvider';
+import { ShowTotalsToggleWrapper } from './ShowTotalsToggle.styles';
+
+const ShowTotalsToggle: FC = () => {
+    const {
+        tableConfig: { showColumnCalculation, setShowColumnCalculation },
+    } = useVisualizationContext();
+    return (
+        <ShowTotalsToggleWrapper>
+            <Label>Show column total</Label>
+            <Switch
```

Comment:
Should we use the `label` + `alignIndicator ` attributes from the `Switch` component ? I think it would align better and match other places in the app.